package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "uid2-sdk",
-  "version": "3.0.1",
+  "name": "@uid2/uid2-sdk",
+  "version": "3.0.3",
```

Comment:
is this the version of this sdk just wondering why it jump from .1 to .3?

Reply:
The version was already set to 3.0.3 in `package.json`, it seems `package.json` and `package-lock.json` had gotten out of sync. When I `npm install`ed webpack-dev-server, it brought the versions (and the package name) in sync.

@lionell-pack-ttd should I bump the version as part of this PR?